index.html

```diff
@@ -41,25 +41,9 @@
     license: 'w3c-software-doc',
     wgPublicList: "public-web-perf",
     noLegacyStyle: true,
+    github: "https://github.com/w3c/beacon",
+    testSuiteURI: "https://github.com/w3c/web-platform-tests/tree/master/beacon",
```

Comment:
Please use:
https://w3c-test.org/beacon/

Otherwise, it's painful for people to run the tests without cloning the tests, etc.